src/hooks/v1/Projects.ts

```diff
@@ -118,6 +126,56 @@ export function useProjectsSearch(handle: string | undefined) {
   )
 }
 
+export function useTrendingProjectsCache() {
```

Comment:
i hadn't tried to extract the cache functionality since i think for now Trending is the only place it makes sense to use.

Using it for All Projects would be awesome, though my two concerns are:
1) When a user creates a new project, it should show in the list immediately
2) The All Projects list is infinite (vs the finite Trending), so we'd either be cacheing a potentially massive piece of data, or cacheing only the first page—with some arbitrary filter applied—which would require more complexity to handle. But I'm open to working on some version of that!

...all that said, i went and extracted it anyway for the sake of code organization lol. will be pushing another update shortly

Reply:
oh shit!! beast, I was just noting a passing thought for the future 😅 ❤️ Thats awesome though.